### PR TITLE
Improve round-start TEG on Marathon Station

### PR DESCRIPTION
**Pull request by Partmedia**
## About the PR
Improves the round-start piping to encourage more use of the TEG on Marathon Station.

## Why / Balance
The TEG was added, but there are still a lot of engineering players that don't set it up either because it's too much work, or because of skill issue (not knowing how to operate a TEG). This problem is particularly amplified on Marathon where the TEG setup left my head scratching. [OK'd by emisse](https://discord.com/channels/310555209753690112/771041304108072961/1151774961401348146).

This PR changes the round-start TEG layout to hopefully make using the TEG a bit easier. It encourages use of the atmos burn chamber, and then piping the burn over to TEG, where it is spent and exhausted.

Despite this, operating the TEG still requires some technical know-how, because with this setup, you still must:

1. Set up pipes and mixers to create a burn mix.
2. Turn on the pump and light the chamber.
3. Charge the coolant loop with a coolant.
4. Turn on the coolant pump.
5. Open the exhaust blast doors.
6. Turn on the hot end valve.

marathon-teg

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Partmedia** [comment]
> This makes it even less accessible to regular engineers as all the input connectors for burn chamber gases were removed.
> 
> Those leftover radiators inside the burn chamber seem out of place, they are not even connected and it's not how radiators work in this game, they end in a pipe cap.

I can remove them if you want. These weren't functional before.

> I assume you removed the pipes under the plasma windows, meaning anyone wanting to utilize either of the chambers in TEG room will need to first dismantle the windows.

Do not assume. The pipes for the radiators are still there.

> This setup is also entirely against how TEG itself is designed, by which I mean that TEG regulates how much energy it generates based on network power draw. Here it just goes straight into space without any sort of limit.

This claim is not true. While it is true that the TEG only extracts as much power from the gas as needed, that does not make this design against "entirely against how TEG itself is designed".

In this setup, it is true that unextracted energy is wasted by ejecting it out to space. The intended way for you to waste less fuel is, you guessed it, having atmos pump less plasma into the burn chamber.

> I'm not sure how this is meant to be easier to setup than the previous layout, as it's now only accessible to atmos techs without a full rebuild. The atmos burn chamber will be used for making tritium and dismantled every other shift, making it even more of a hassle to setup TEG for the laymen.

These concerns seem to stem out of a mindset of using the old layout. The old layout used pumps on the hot end, which artificially limit the transfer rate through the hot end and cap the power output. The old layout's pumps also choke the exhaust from the burn chamber, causing the burn to sometimes go in and out. This new design improves this on all fronts.

In short, I think you should give this setup a try in game instead of just looking at the picture and thinking "this looks completely different from the setup I understand, it must be wrong."

**Comment by Partmedia** [comment]
> There's only 1 burn chamber for atmospherics. There's 2 "burn chambers" for TEG. Your change turned the 1 burn chamber atmospherics used for tritium, into another burn chamber for TEG while making the previous 2 chambers obsolete leftovers.

That is correct. I can remove the previous two unused obsolete leftovers if you prefer. Why does atmos need a separate burn chamber to produce tritium? Why can't they produce tritium *and* feed the TEG using the same burn?

**Comment by Emisse** [comment]
theres no real hot loop here since the fuel line just feeds into it and it exhausts into the chamber

**Comment by Partmedia** [comment]
> theres no real hot loop here since the fuel line just feeds into it and it exhausts into the chamber

Why does there need to be a hot "loop"? You're intended to try to extract all of the heat out of the gas in one go, then vent it out. If you don't need that much power, reduce the plasma flow into the burn chamber. Gas is always flowing, despite there not being a loop, by virtue of the burn chamber producing more gas.

**Comment by deltanedas** [comment]
if you are doing a teg setup like that always just pump it into a lattice tile theres no reason to keep it around

inb4 gas miners will go away, then these setups will finally die and we will use chambers and radiators properly

**Comment by Partmedia** [comment]
Moved burn chamber into TEG room.

image

**Comment by Partmedia** [comment]
Overcome by events.
